translation/migrations/0001_initial.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-07-11 12:09
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('api_controller', '0001_initial'),
```

Comment:
@KirillMakarov yeah I need to delete all migrations and make meaningfull names, as there is no safe way to delete migrations